airbyte-integrations/bases/standard-destination-test/src/main/resources/edge_case_messages.txt

```diff
@@ -7,4 +7,19 @@
 {"type": "RECORD", "record": {"stream": "reserved_keywords", "emitted_at": 1602637589000, "data": { "order" : "ascending" }}}
 {"type": "RECORD", "record": {"stream": "groups", "emitted_at": 1602637589000, "data": { "authorization" : "into" }}}
 {"type": "RECORD", "record": {"stream": "ProperCase", "emitted_at": 1602637589000, "data": { "ProperCase" : true }}}
+{"type": "RECORD", "record": {"stream": "stream_with_dates", "emitted_at": 1602637589000, "data": { "some_date" : "2021-01-01", "some_datetime" : "2021-01-01T01:01:01-01:00" }}}
```

Comment:
it may be clearer to have each record have exactly one of the fields, this way it's clear what each record is testing.

Reply:
Done.